contrib/RL_out_sharp.lua

```diff
@@ -0,0 +1,208 @@
+--[[
+
+  DESCRIPTION
+    RL_out_sharp.lua - Richardson-Lucy output sharpening using GMic
+
+    This script provides a new target storage "RL output sharpen". 
+    Images exported will be sharpened using GMic (RL deblur algorithm)
+    
+  USAGE
+    * require this script from main lua file
+    * in lua preferences, select the GMic cli executable
+    * from "export selected", choose "RL output sharpen"
+    * configure output folder 
+    * configure RL parameters with sliders
+    * configure temp files format and quality, jpg 8bpp (good quality) 
+      and tif 16bpp (best quality) are supported
+    * configure other export options (size, etc.)
+    * export, images will be first exported in the temp format, then sharpened
+    * sharpened images will be stored in jpg format in the output folder
+
+  EXAMPLE
+    set sigma = 0.7, iterations = 10, jpeg output quality = 95,
+    that will be sufficient for most uses
+
+  CAVEATS
+    MAC compatibility not tested
+  
+  BUGS, COMMENTS, SUGGESTIONS
+    send to Marco Carrarini, marco.carrarini@gmail.com
+
+  CHANGES
+    * 20200308 - initial version
+  
+]]
+
+local dt = require "darktable"
+local du = require "lib/dtutils"
+local df = require "lib/dtutils.file"
+local dtsys = require "lib/dtutils.system"
+
+-- module name
+local MODULE_NAME = "RL_out_sharp"
+
+-- check API version
+du.check_min_api_version("5.0.2", MODULE_NAME)  -- darktable 3.x
```

Comment:
This should be "5.0.0".  API versions 5.0.1 and 5.0.2 introduced some functions to control panel visibility, darkroom image selected, and collection manipulations.  Since you're not using any of those things, you the minimum API version is 5.0.0 since you are using dt.configuration.running_os.  This way the script will be compatible with darktable 2.6.x, for those who haven't upgraded yet.

Reply:
done